### ISSUE TITLE
Any potential for Other titles like AC Valhalla?

### ISSUE THREAD

**Issue by khatanaashish** [tuffee88/d3d12ProxyEdrDx11_0#8]
After testing your fix, I am hopeful for other titles. We saw something similar for Death Stranding. Your logic is very simple and efficient. Maybe other games are doable in a similar fashion, for ex: Anvil engine's AC Valhalla which also runs on older PS4. 🤔

**Comment by tuffee88** [comment]
Could be, but this is a rather long shot - the reason this works for Elden Ring is that the game doesn't really use any features that are only available for Feature Level 12_0 and also doesn't query/check the features supported

Tbh there might be some missing or different bits and pieces but overall the game seems to run and mostly look the same on older cards - however they might also support features that were optional for Feature Level 11_0 but became mandatory for 12_0 so the actual behaviour is likely different depending on exact GPU.

To check if this is the same situation for AC Valhall you can just follow the same instructions as for Elden Ring - if that's really the same situation this technique should just work. In any case please let me know, AC Valhalla is a game I might pick up later this year :-)

**Comment by mojmass** [comment]
I can confirm that this works on Assassin's Creed Valhalla using GTX 850m.
Works like a charm, thank you so much.
I wonder if something similar can be done for God Of War which requires feature level 11_1, which my graphic card supports (says so on dxdiag) but still get the error feature level not supported. There is a bypass by using DXVK but the performance is not ideal

**Comment by mstiya** [comment]
Hey hey.

I can confirm that this works on Death Stranding using GTX 750ti, if it matters.

**Comment by tuffee88** [comment]
Hi,
glad to hear that - thanks for letting me know, I've added a note to the README !

**Comment by SetoAce** [comment]
I wonder if this fix will work on Splinter Cell Blacklist. IIRC a lot of people saying that its Dx12 is mostly the cause of CTD.

**Comment by tuffee88** [comment]
> I wonder if this fix will work on Splinter Cell Blacklist. IIRC a lot of people saying that its Dx12 is mostly the cause of CTD, even if the game allows you to switch to Dx11 .

Maybe, maybe not, feel free to give this a short if you're interested the procedure should be the same as with Elden Ring, ecc.  :-)

**Comment by FilipSzweda** [comment]
The fix doesn't work on Death Stranding when using GTX 770. Do you recon if it's possible to get it running in the scope of this project or is this game a whole different story than Elden Ring?
[error.txt](https://github.com/tuffee88/d3d12ProxyEdrDx11_0/files/8355378/error.txt)

**Comment by tuffee88** [comment]
> The fix doesn't work on Death Stranding when using GTX 770. Do you recon if it's possible to get the game running in the scope of this project or is it a whole different story than Elden Ring? [error.txt](https://github.com/tuffee88/d3d12ProxyEdrDx11_0/files/8355378/error.txt)

Maybe, can you provide the log from DebugView ? Did you try both the release and debug binaries ?

**Comment by FilipSzweda** [comment]
I'm sending the log 👍 
I've tried both the release and debug binaries, but none of them seems to work.
[DESKTOP-I3F5AFU.LOG](https://github.com/tuffee88/d3d12ProxyEdrDx11_0/files/8358398/DESKTOP-I3F5AFU.LOG)

**Comment by Gaming1992** [comment]
Same issue,force close after 5 minute,i check debug view get message d3d12: Removing device,does anyone know how to fix this problem on AC valhalla?

**Comment by tuffee88** [comment]
> I'm sending the log 👍 I've tried both the release and debug binaries, but none of them seems to work. [DESKTOP-I3F5AFU.LOG](https://github.com/tuffee88/d3d12ProxyEdrDx11_0/files/8358398/DESKTOP-I3F5AFU.LOG)

Thanks for the log ! From what I can see in the log entries D3D12 init is called twice (fails the first time and succceeds the second time). I'll have a look and check if I can reproduce this on my card (Nvidia Kepler too).

**Comment by tuffee88** [comment]
> Assasins creed valhalla ,force close after 5 minute,on my gtx 750 ti,i check debug view get message d3d12: Removing device,does anyone know how to fix this problem on AC valhalla?

None that I know of - some people report that you can get more time until it crashes when you lower the settings (+resolution).

**Comment by Gaming1992** [comment]
I think the problem lies in the game, in the ac valhalla folder, the name is shaderdatadx12.bin this needs to be modified, maybe you can do it

**Comment by GorkemKurter** [comment]
Anybody try to run Vallhalla with this fix on 930m?

**Comment by FilipSzweda** [comment]
> > I'm sending the log 👍 I've tried both the release and debug binaries, but none of them seems to work. [DESKTOP-I3F5AFU.LOG](https://github.com/tuffee88/d3d12ProxyEdrDx11_0/files/8358398/DESKTOP-I3F5AFU.LOG)
> 
> Thanks for the log ! From what I can see in the log entries D3D12 init is called twice (fails the first time and succceeds the second time). I'll have a look and check if I can reproduce this on my card (Nvidia Kepler too).

Hey! :) Have you maybe uncovered what was the issue? Is it perhaps Kepler architecture hardware limitations towards Death Stranding i.e. the game won't run without modifications?

**Comment by tuffee88** [comment]
Not quite, the repeated call to D3D12 init is benign (behaviour is documented in MSDN and used to check if device creation would succeed) - why Death Stranding doesn't work is still a mystery to me.

 For sure it has to do with missing support (HW or Driver) but I couldn't pinpoint the exact trigger that causes the crash, if somebody can provide more information on that we might figure out if a workaround is possible, but as it is now I don't think we'll be able to get this running.

**Comment by rafaelvcarelli** [comment]
Any chances we get to run FIFA 23? It won’t run on video drivers that don’t have Feature Level 12.

**Comment by moehammm** [comment]
[284]LogRunningAppService: Display: New foreground app D:/Program Files (x86)/FIFA 23/ 
[ 59]LogRunningAppService: Display: **Dead** foreground process [13588] D:/Program Files (x86)/FIFA 23/EAAntiCheat.GameServiceLauncher.exe for app D:/Program Files (x86)/FIFA 23/ 
[ 59]LogRunningAppService: Display: New foreground process [10864] D:/Program Files (x86)/FIFA 23/FIFA23.exe for app D:/Program Files (x86)/FIFA 23/ 
[344]LogEOSMessageService: Error: **Unable to find port**. 
[ 56]LogPortalPublishTime: Create playtime timespan. Item: 

This is FIFA 23 on quadro m1000m 
Did not work, latest drivers installed.

**Comment by clawdius** [comment]
Hi there, I'm coming from the future, I have GTX 960M and using this proxy d3d12.dll, I can get Helldivers 2 to works. It has an anti cheat system and so far after 33 hours playing, I'm fine.

The thing is, it's kind of complicated to get it to work, since I need to use other workaround because Helldivers 2 can't even recognize this graphic card. I had to use **dxvk** and **vkd3d**.

**Comment by Soup7674** [comment]
Used this to run the newly released GTA V Enhanced for PC (which required feature level 12_0) on an Nvidia Quadro k2100m (API level dx12, feature level 11_0), ran flawlessly with no noticeable bugs

**Comment by whyamiinaussie** [comment]
> Used this to run the newly released GTA V Enhanced for PC (which required feature level 12_0) on an Nvidia Quadro k2100m (API level dx12, feature level 11_0), ran flawlessly with no noticeable bugs

what did u do?

**Comment by clawdius** [comment]
Reporting in for another supported titles, I can confirm to get **Kingdom Come Deliverance II** working on my GTX 960m with this proxy again, cheers!
